### ISSUE TITLE
Add Reliability KR to Product Health MVP

### ISSUE THREAD

**Issue by rroueche** [department-of-veterans-affairs/va.gov-team#1030]
## User Story
As the VSP Team, I need to understand if we're hitting our Reliability KRs, so I can determine whether we need to prioritize any changes to our product and the way it serves both VSP users and VA.gov users.

## Goal
Update MVP Product Health tool to display whether we are hitting our goal of constantly having 90% of WBC-based page loads occur in <5s. Found by adding up bucket percentages above 5s: https://analytics.google.com/analytics/web/?utm_source=marketingplatform.google.com&utm_medium=et&utm_campaign=marketingplatform.google.com%2Fabout%2Fanalytics%2F#/report/content-site-speed/a50123418w177519031p184624291/_u.date00=20190601&_u.date01=20190630&_r.tabId=histogramTab&histogramTab-advancedHistogramTable.open=3000/

## Acceptance Criteria
- [ ] Recommend best way to display this (trend over time? point in time?) cc: @amycesal 
- [ ] Product Health MVP displays this in appropriate place
- [ ] It is automated to update in realtime, so we don't have to pull manually from GA cc: @jonwehausen

**Comment by joanneesteban** [comment]
@nedierecel @jonwehausen Where are we with this issue?

**Comment by nedierecel** [comment]
@jonwehausen @joanneesteban Using the existing method of Google Sheets, the Reporting API is outputting the total page time and not the distribution (as we see in the Google Analytics interface).

Here is my test, so I don't muck up the [existing prototypes](https://docs.google.com/spreadsheets/d/1UOHXIIIHLuUCURbJPzK3WpXh3pH90of9FPt9Zs2yuaA/edit#gid=681579896).

Jon, I used `ga:pageLoadTime` and `ga:pageLoadSample`, with the expectation that it would've provided the same type of data as the [Page Load Time report in GA](https://analytics.google.com/analytics/web/?utm_source=marketingplatform.google.com&utm_medium=et&utm_campaign=marketingplatform.google.com%2Fabout%2Fanalytics%2F#/report/content-site-speed/a50123418w177519031p184624291/_u.date00=20190601&_u.date01=20190630&_r.tabId=histogramTab&histogramTab-advancedHistogramTable.open=3000/)

**Comment by joanneesteban** [comment]
Will need API requests to get the data that is required in correct formatting. @joanneesteban follow up with Rachael to see priority on this.

**Comment by joanneesteban** [comment]
@jonwehausen @nedierecel This should be a bi-quarterly update - if it's easier, a monthly update.

**Comment by jonwehausen** [comment]
CC: @joanneesteban @nedierecel @rroueche 

Hi Team, 

I spent some time researching how we would do this programmatically thru the API. Unfortunately, the API is very limited in the ability to extract / output data in a comparable format to [this type of report](https://analytics.google.com/analytics/web/?utm_source=marketingplatform.google.com&utm_medium=et&utm_campaign=marketingplatform.google.com%2Fabout%2Fanalytics%2F#/report/content-site-speed/a50123418w177519031p184624291/_u.date00=20190601&_u.date01=20190630&_r.tabId=histogramTab&histogramTab-advancedHistogramTable.open=3000/) (_GA calls this a histogram_). 

If we did want to leverage GA for this data thru the API, it would require the better part of a full sprint as if would involve custom timer bucketing thru Javascript to use as a custom dimension. 

I think this request is also a strong candidate for a Grafana dashboard -- given that GA isn't a great source for performance based metrics as page load time as-is. 

Look forward to hearing y'alls thoughts.
